qingcloud/qingstor/bucket.py

```diff
@@ -46,6 +45,18 @@ def __getitem__(self, key_name):
     def __len__(self):
         pass
 
+    def load_data(self, response_data):
+        """ Judge the type of data, fix the json.loads error.
```

Comment:
The docstring looks unrelated with the function's purpose. 

"""Wrapper to load json data, to be compatible with Python3
....
...
"""